src/Exception/RequestException.php

```diff
@@ -136,13 +139,18 @@ public static function getResponseBodySummary(ResponseInterface $response)
             return null;
         }
 
-        $summary = $body->read(120);
-        $body->rewind();
+        if (self::$responseSize) {
+            $summary = $body->read(self::$responseSize);
+        } else {
+            $summary = $body->getContents();
+        }
 
-        if ($size > 120) {
+        if (self::$responseSize && $size > self::$responseSize) {
             $summary .= ' (truncated...)';
         }
 
+        $body->rewind();
```

Comment:
Why should we rewind it? Maybe it was not rewinded form the start..

Reply:
It's been some time, I don't fully remember why I did the `rewind`. Removed.